nixos/modules/security/gnupg.nix

```diff
@@ -0,0 +1,270 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (builtins) attrNames dirOf length match split;
+  inherit (lib) types;
+  cfg = config.security.gnupg;
+  gnupgHome = "/var/lib/gnupg";
+  # Escape as required by: https://www.freedesktop.org/software/systemd/man/systemd.unit.html
+  escapeUnitName = name:
+    lib.concatMapStrings (s: if lib.isList s then "-" else s)
+    (split "[^a-zA-Z0-9_.\\-]+" name);
+in
+{
+options.security.gnupg = {
+  store = lib.mkOption {
+    type = with types; either str path;
+    default = "/root/.password-store";
+    description = ''
+      Default base path for the <literal>gpg</literal> option
+      of the <link linkend="opt-security.gnupg.secrets">secrets</link>.
+      Note that you may set it up with something like:
+      <literal>builtins.getEnv "PASSWORD_STORE_DIR" + "/machines/example"</literal>.
+    '';
+  };
+  secrets = lib.mkOption {
```

Comment:
I typically find it improves readability and navigability if there is an empty line between option definitions.